docs/setup-and-running.md

```diff
@@ -0,0 +1,96 @@
+## Setup and Running
+
+It is highly recommended to use a Unix-based system (such as Ubuntu, Fedora, Kali, or any other) or MacOS. Windows may encounter certain issues with Docker.
+
+### Requirements
+
+- Git is essential before anything else;
+- [Node][1] and [TypeScript][2];
+- If you are on MacOs, you can install above using [homebrew][3];
+- [Docker][4]
+- [pnpm][5]
+- [Prisma][6]
+- [GPG][7]
+- [PostGres][8]
+
+## 1 - Clone the Repository
+
+To clone the repository, follow these steps:
+
+Open your terminal or command prompt.
+
+Go to the file that you want to download the project (you can use the command `ls` to see where your location and the comand `cd` to go to myfiles/exemple).
+
+Then, run the following command to clone the repository:
+
+`git clone https://github.com/podcodar/app.git` \
+This command will download the project to your local machine. ✨
+
```

Comment:
I think you can add a full enter here, to have this line break in the final view. It looks better to me, but please let me know what you think!

```suggestion
`git clone https://github.com/podcodar/app.git` 

This command will download the project files to your local machine in the folder `app`. ✨

```

Also added a small suggestion in the sentence

Reply:
I liked the suggestion. I think it looks better visually.